src/components/ConnectToSageIntacctFlow/index.tsx

```diff
@@ -58,56 +55,54 @@ function ConnectToSageIntacctButton({policyID, shouldDisconnectIntegrationBefore
         },
     ];
 
+    useEffect(() => {
+        if (!isControlPolicy(policy)) {
+            Navigation.navigate(ROUTES.WORKSPACE_UPGRADE.getRoute(policyID, CONST.UPGRADE_FEATURE_INTRO_MAPPING.intacct.alias));
```

Comment:
```suggestion
            Navigation.navigate(
                            ROUTES.WORKSPACE_UPGRADE.getRoute(
                                policyID,
                                CONST.UPGRADE_FEATURE_INTRO_MAPPING.intacct.alias,
                                ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_PREREQUISITES.getRoute(policyID),
                            ),
                        );
```

Reply:
Same reason as above

Reply:
and done :)

Reply:
oh wait, I didn't see this changes, can you verify? @war-in

Reply:
and the previous comment too, I think you forgot to push your changes?

Reply:
You're right, we moved it to another file 😅 Let me fix that

Reply:
I noticed that the linked PR is reverted. We probably shouldn't add those changes here

https://github.com/Expensify/App/pull/46460

Reply:
Damn 😂. Okay, let keep it as it is haha

Reply:
However, I pushed changes: https://github.com/Expensify/App/pull/45251/commits/57b0addcfd625628160a422ed67f753b91ef1d28 because we missed intacct navigation there!

So thank you for noticing 🙇